docs/mdbook/default.nix

```diff
@@ -6,6 +6,8 @@
   nixosOptionsDoc,
   transformOptions,
   hmOptions,
+  # The root directory of the site
+  baseHref ? "/",
```

Comment:
I guess we want to merge this somewhat in-sync with #2537, as otherwise this could mess up relative links on the stable docs?